test-browser/tests/pluginManager.js

```diff
@@ -0,0 +1,129 @@
+     .click('*[data-id="pluginManagerComponentSearchInput"]')
+     .keys('debugger')
+     .waitForElementVisible('*[data-id="pluginManagerComponentActivateButtondebugger"]')
+     .clearValue('*[data-id="pluginManagerComponentSearchInput"]')
+     .click('*[data-id="pluginManagerComponentSearchInput"]')
+     .keys('Deploy & run transactions')
+     .waitForElementVisible('*[data-id="pluginManagerComponentActivateButtonudapp"]')
+     .clearValue('*[data-id="pluginManagerComponentSearchInput"]')
+     .click('*[data-id="pluginManagerComponentSearchInput"]')
+     .keys('ZoKrates')
+     .waitForElementVisible('*[data-id="pluginManagerComponentActivateButtonZoKrates"]')
+     .clearValue('*[data-id="pluginManagerComponentSearchInput"]')
+     .click('*[data-id="pluginManagerComponentSearchInput"]')
+     .keys(browser.Keys.ENTER)
+  },
+
+  'Should activate plugins': function (browser) {
+    browser.waitForElementVisible('*[data-id="pluginManagerComponentPluginManager"]')
+    .click('*[data-id="pluginManagerComponentPluginManager"]')
+    .scrollAndClick('*[data-id="pluginManagerComponentActivateButtondebugger"]')
+    .waitForElementVisible('*[data-id="pluginManagerComponentDeactivateButtondebugger"]')
+    .scrollAndClick('*[data-id="pluginManagerComponentActivateButtonudapp"]')
+    .waitForElementVisible('*[data-id="pluginManagerComponentDeactivateButtonudapp"]')
+    .scrollAndClick('*[data-id="pluginManagerComponentActivateButtonZoKrates"]')
+    .waitForElementVisible('*[data-id="pluginManagerComponentDeactivateButtonZoKrates"]')
+  },
+
+  'Should deactivate plugins': function (browser) {
+    browser.waitForElementVisible('*[data-id="pluginManagerComponentPluginManager"]')
+    .click('*[data-id="pluginManagerComponentPluginManager"]')
+    .scrollAndClick('*[data-id="pluginManagerComponentDeactivateButtondebugger"]')
+    .waitForElementVisible('*[data-id="pluginManagerComponentActivateButtondebugger"]')
+    .scrollAndClick('*[data-id="pluginManagerComponentDeactivateButtonudapp"]')
+    .waitForElementVisible('*[data-id="pluginManagerComponentActivateButtonudapp"]')
+  },
+
+  'Should grant plugin permission (ZOKRATES)': function (browser) {
+    browser.waitForElementVisible('*[data-id="pluginManagerComponentPluginManager"]')
+    .click('*[data-id="pluginManagerSettingsButton"]')
+    .waitForElementVisible('*[data-id="pluginManagerSettingsPermissionForm"]')
+    .assert.containsText('*[data-id="pluginManagerSettingsPermissionForm"]', 'No Permission requested yet')
+    .modalFooterOKClick()
+    .click('*[data-id="verticalIconsFileExplorerIcons"]')
+    .switchFile('browser/3_Ballot.sol')
+    .click('*[plugin="ZoKrates"]')
+    .pause(5000)
+    .frame(0)
+    .useXpath().click("//span[text()='Compile']")
+    .frameParent()
+    .useCss().waitForElementVisible('*[data-id="modalDialogContainer"]')
+    .assert.containsText('*[data-id="permissionHandlerMessage"]', 'ZOKRATES" WOULD LIKE TO ACCESS "FILE MANAGER" :')
```

Comment:
we should also check is the api call is successful and returns the correct data, but that will be another PR...